README.md

```diff
@@ -1 +1,62 @@
-# bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# -Bookstore  <img src="https://img.icons8.com/external-filled-outline-wichaiwi/344/external-description-e-commerce-website-filled-outline-wichaiwi.png" alt="Bootstrap Icon" style="width: 50px; height: 50px">
+
```

Comment:
- [ ] (Optional) It would be nice to remove the hyphen `(-)` before the `Bookstore`. This would make your `README.md` look more professional and it would show you are detailed and watch out for typographical errors. 👍